glide-loader/src/main/java/org/wordpress/aztec/glideloader/extensions/BitmapExtension.kt

```diff
@@ -0,0 +1,10 @@
+package org.wordpress.aztec.glideloader.extensions
+
+import android.graphics.Bitmap
+import android.graphics.Bitmap.createScaledBitmap
+
+internal fun Bitmap.upscaleTo(desiredWidth: Int): Bitmap {
+    val ratio = this.height / this.width
```

Comment:
I've just found problems when resizing pictures whose h/w ratio is less than 1.
We should make sure to use Float variables when calculating the `ratio`, and then cast to int at the end of the routine. Something like the following.

``` 
val ratio : Float = (this.height * 1f) / this.width
val proportionateHeight : Float = ratio * desiredWidth
val finalHeight = Math.rint(proportionateHeight.toDouble()).toInt()

return createScaledBitmap(this, desiredWidth, finalHeight, true)
```

We could also add an additional check on `finalHeight`, ensuring that it's `> 0`

Reply:
Good catch. Original code from wpandroid used floats, but I brainfarted and removed the casts for some reason :) 

Rounding is a nice touch too!